pyproject.toml

```diff
@@ -20,6 +20,7 @@ dependencies = [
     "amaranth~=0.4.0",
     "amaranth-boards @ git+https://github.com/amaranth-lang/amaranth-boards.git@main",
     "usb-protocol @ git+https://github.com/usb-tools/python-usb-protocol",
+    "cynthion @ git+https://github.com/greatscottgadgets/cynthion/#subdirectory=cynthion/python/"
```

Comment:
I don't think this should become a required dependency, it should either go in the optional dependencies or the CI script should install it manually.

Reply:
Great point!